data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, weight_kg, neutered ) VALUES (1,'Agumon', '02/03/2020',0, 10.23, true);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, weight_kg, neutered ) VALUES (2,'Gabumon', '10/15/2022', 2, 15.04, false);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, weight_kg, neutered ) VALUES (3,'Pikachu', '01/13/2020', 1, 0.5, false);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, weight_kg, neutered ) VALUES (4,'Devimon', '05/04/2017', 5, 11.13, true);
```

Comment:
- [ ] The data in some columns is incorrect. Please make sure you correct all the data according to the project requirements.

![image](https://user-images.githubusercontent.com/83803414/212922101-07b27614-60f8-454c-9097-16519c8e6bfc.png)